cpg_workflows/jobs/genotype.py

```diff
@@ -322,7 +322,11 @@ def postproc_gvcf(
     # meaning we have more than enough disk (265/8=33.125G).
     # Enough to fit a pre-reblocked GVCF, which can be as big as 10G,
     # the reblocked result (1G), and ref data (5G).
-    job_res = STANDARD.set_resources(j, ncpu=2, storage_gb=20)
+    if (postproc_gvcf_storage := get_config()['resource_overrides'].get('postproc_gvcf_storage')) is not None:
```

Comment:
Slightly rusty, so ignore me if I'm off here, but I believe this can be simplified to
`storage_gb = get_config()['resource_overrides'].get('postproc_gvcf_storage', 20)`?

(Similar to the scatter_count construction in line 47)

Reply:
Thanks Katie, this is cleaner that what I had